File: python/src/Behavioral/Strategy/DvdNameReplaceSpacesStrategy.py
#//DvdNameReplaceSpacesStrategy.java - three of three concrete strategies


from DvdNameStrategy import DvdNameStrategy

class DvdNameReplaceSpacesStrategy (DvdNameStrategy) :

	def formatDvdName (self, dvdName, charIn) :
		return dvdName.replace(' ',charIn) ## should be replace all, not replace one char

File: ruby/src/Behavioral/Strategy/DvdNameReplaceSpacesStrategy.rb
#//DvdNameReplaceSpacesStrategy - three of three concrete strategies


#require "Behavioral/Strategy/DvdNameStrategy"

class DvdNameReplaceSpacesStrategy < DvdNameStrategy

	def initialize
	end

	def formatDvdName ( dvdName, charIn)
 		return dvdName.gsub(' ',charIn) ## should be replace all, not replace one char
	end
end

